<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Letture</class>
 <widget class="QDialog" name="Letture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Letture</string>
  </property>
  <property name="locale">
   <locale language="Italian" country="Italy"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>481</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>70</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_L">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Indietro</string>
        </property>
        <property name="locale">
         <locale language="Italian" country="Italy"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/back-button.ico</normaloff>:/back-button.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Calligraphy</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Letture</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="LettureTable">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>250</height>
         </size>
        </property>
        <property name="locale">
         <locale language="Italian" country="Italy"/>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>ISBN</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Titolo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Genere</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Pagine</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Prezzo</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Disp.</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cop. Flex.</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButton_NewL">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Nuovo libro</string>
            </property>
            <property name="toolTipDuration">
             <number>-1</number>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="locale">
             <locale language="Italian" country="Italy"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/addBook.ico</normaloff>:/addBook.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="pushButton_DelL">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Elimina libro</string>
            </property>
            <property name="toolTipDuration">
             <number>-1</number>
            </property>
            <property name="locale">
             <locale language="Italian" country="Italy"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/cancelBook.ico</normaloff>:/cancelBook.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_AddL">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Inserisci libro</string>
            </property>
            <property name="toolTipDuration">
             <number>-1</number>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="locale">
             <locale language="Italian" country="Italy"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/InsertBook.ico</normaloff>:/InsertBook.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_RemL">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Rimuovi libro</string>
            </property>
            <property name="toolTipDuration">
             <number>-1</number>
            </property>
            <property name="locale">
             <locale language="Italian" country="Italy"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/BookRemove.ico</normaloff>:/BookRemove.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButton_EditL">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Modifica libro</string>
            </property>
            <property name="toolTipDuration">
             <number>-1</number>
            </property>
            <property name="locale">
             <locale language="Italian" country="Italy"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/EditBook.ico</normaloff>:/EditBook.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="locale">
         <locale language="Italian" country="Italy"/>
        </property>
        <property name="text">
         <string>Cerca:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Cerca libro</string>
        </property>
        <property name="locale">
         <locale language="Italian" country="Italy"/>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
